README.md

```diff
@@ -2,7 +2,7 @@
 
 # pgEdge Terraform Provider
 
-The official Terraform provider for [pgEdge](https://www.pgedge.com/), designed to simplify the management of pgEdge resources for both **Developers** and **Enterprise** edition.
+The official Terraform provider for [pgEdge Cloud](https://www.pgedge.com/cloud), designed to simplify the management of pgEdge Cloud resources for both **Developers** and **Enterprise** edition.
```

Comment:
This should be "Developer" not "Developers" - this is referring to Developer Edition